renovate.json

```diff
@@ -9,5 +9,18 @@
   ],
   "extends": [
     "local>brave/renovate-config"
+  ],
+  "packageRules": [
+    {
+      "matchPackageNames": [
+        "aws-sdk"
+      ],
+      "minor": {
```

Comment:
```    {
      ...
      "extends": ["default:automergeMinor"],
      "enabled": true
    },
```

not sure if the "enabled" is required, but you might be able to solve it with only the "extends"

Reply:
Is this what you meant?

Reply:
Or do I need to put both `extends` just under:
```
"matchPackageNames": [
    "aws-sdk"
],
```